OpenSSL-Query/README.md

```diff
@@ -23,6 +23,8 @@ OpenSSL::Query requires these extra modules to run:
 - Class::Method::Modifiers	(debian package libclass-method-modifiers-perl)
 - Moo				(debian package libmoo-perl)
 - URI::Encode			(debian package liburi-encode-perl)
+- LWP::UserAgent
+- LWP::Protocol::https
```

Comment:
Please change the "requires" section in `Makefile.PL` as well

Reply:
Added.
I'm not au fait with Perl.

Reply:
I've updated the required section (I think).